odoo/tools/config.py

```diff
@@ -725,6 +724,10 @@ def __setitem__(self, key, value):
     def __getitem__(self, key):
         return self.options[key]
 
+    @property
+    def root_path(self):
+        return self._normalize(os.path.join(os.path.dirname(__file__), '..'))
```

Comment:
Not sure it is correct, but i would have written that 
`return os.path.dirname(os.path.dirname(__file__)))`

But this may not be correct in all cases (symlink, ...) ?